spec/paperclip/has_attached_file_spec.rb

```diff
@@ -34,31 +34,41 @@
       assert_adding_attachment('avatar').defines_callback('before_destroy')
     end
 
-    it 'defines an after_commit callback' do
+    it 'defines the Paperclip-specific callbacks' do
+      assert_adding_attachment('avatar').defines_callback('define_paperclip_callbacks')
+    end
+
+    it 'defines an after_commit callback if the class allows after_commit callbacks' do
       assert_adding_attachment('avatar').defines_callback('after_commit')
     end
 
-    it 'defines the Paperclip-specific callbacks' do
-      assert_adding_attachment('avatar').defines_callback('define_paperclip_callbacks')
+    it 'defines an after_destroy callback if the class does not allow after_commit callbacks' do
+      assert_adding_attachment('avatar', :unstub_methods => [:after_commit]).defines_callback('after_destroy')
```

Comment:
Line is too long. [110/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.